sql_user_check_sql_server/database_stats.sh

```diff
@@ -2,10 +2,12 @@
 
 mode="stats"
 port="1433"
+use_ad_auth=0
```

Comment:
We need to explain this flag in README

Reply:
Provided the explanation that for AD based authentication, -a 1 needs to be set.